.github/workflows/preview-commit.yml

```diff
@@ -87,5 +87,4 @@ jobs:
         run: ls -R npm
 
       - name: Release
-        if: needs.check-changed.outputs.changed == 'true'
```

Comment:
Does this means this action no longer skip the Release job?

Reply:
Noop, just advanced the skipped judgment to the build job

Reply:
If the release is skipped, it should not be built

Reply:
get👌